gitgud/operations.py

```diff
@@ -498,6 +509,15 @@ def get_copy_mapping(self, non_merges, known_commits):
 
         return mapping
 
+    def get_all_commits(self):
+        yielded = set()
+        for head in self.repo.heads:
+            for commit in self.repo.iter_commits(head, reverse=True):
```

Comment:
This gets us all the branches, but it will miss any commits if there's a detached head or tags where the commits aren't present. You should also add a `sorted` flag in case you want the commits sorted (which I think you do)